aspnetcore/security/authentication/accconfirm.md

```diff
@@ -159,9 +261,13 @@ In this section, run the web app and show the account confirmation and password
 If you can't get email working:
 
 * Review the [Email Activity](https://sendgrid.com/docs/User_Guide/email_activity.html) page.
+* Check your spam folder.
+* Try another email alias on a different email provider (Microsoft, Yahoo, Gmail, etc.)
 * Create a [console app to send email](https://sendgrid.com/docs/Integrate/Code_Examples/v2_Mail/csharp.html).
 * Try sending to different email accounts.
 
+Note: A security best practice is to not use production secrets in test and development. If you publish the app to Azure, you can set the SendGrid secrets as application settings in the Azure Web App portal. The configuration system is setup to read keys from environment variables.
```

Comment:
Should this Note use the Note UI widget? If not, should "Note:" be bolded to draw more attention to this text?